xayn-ai/src/coi/mod.rs

```diff
@@ -42,23 +40,13 @@ impl CoiId {
 }
 
 #[derive(Debug, Display, Error)]
-#[allow(clippy::enum_variant_names)]
 pub(crate) enum CoiError {
     /// A key phrase is empty
     EmptyKeyPhrase,
     /// A key phrase has non-finite embedding values: {0:#?}
     NonFiniteKeyPhrase(ArcEmbedding),
     /// A computed relevance score isn't finite.
     NonFiniteRelevance,
-    /// Invalid coi shift factor, expected value from the unit interval
-    InvalidShiftFactor,
-    /// Invalid coi threshold, expected non-negative value
-    #[cfg(test)]
-    InvalidThreshold,
-    /// Invalid coi neighbors, expected positive value
-    InvalidNeighbors,
-    /// Invalid coi gamma, expected value from the unit interval
-    InvalidGamma,
-    /// Invalid coi penalty, expected non-empty, finite and sorted values
-    InvalidPenalty,
+    /// Invalid configuration: {0}
+    InvalidConfiguration(#[from] ConfigError),
```

Comment:
Do you want to use that later? because it is not used anywhere

Reply:
good catch, no it's going to be unused, i'll remove it.